pkg/operator/ceph/nfs/nfs.go

```diff
@@ -303,12 +303,13 @@ func (r *ReconcileCephNFS) configureNFSPool(n *cephv1.CephNFS) error {
 	poolName := n.Spec.RADOS.Pool
 	logger.Infof("configuring pool %q for nfs", poolName)
 
-	_, err := cephclient.GetPoolDetails(r.context, r.clusterInfo, poolName)
+	args := []string{"osd", "pool", "create", poolName}
+	output, err := cephclient.NewCephCommand(r.context, r.clusterInfo, args).Run()
```

Comment:
This is idempotent with no error if the pool is already created? Will it impact anything if they have created the .nfs CephBlockPool CR that has settings other than the defaults?

Reply:
I did a test just to make sure, and things seem to be working safely. 

If I create the `.nfs` CephBlockPool, it keeps those settings when the CephNFS is reconciled. 

If I allow Rook to create the `.nfs` pool itself, it succeeds on a re-reconcile. 

If I then create the `.nfs` CephBlockPool afterwards, the pool gets updated (I changed the failure doamain), but it doesn't revert when the CephNFS is reconciled after that.